docs/api/pet.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Pet
+nav_order: 3
+---
+
+# `pet` resource
+
+Base endpoint:
+
+```shell
+{base_url}/pets
+```
+
+Contains information about pets in the service.
+
+To adopt a pet in the service, the user must first be added to the service. Learn more about the [user resource](user.md).
+
+## Resource properties
+
+Sample `pet` resource:
+
+```js
+{
+    "id": 4,
+    "name": "Fred",
+    "type": "dog",
+    "breed": "Poodle",
+    "age": "2 years",
+    "color": "brown",
+    "personality": "friendly",
+    "status": "available",
+    "user_id": null
+}
+```
+
+| Property name | Type | Description |
+| ------------- | ----------- | ----------- |
+| `id` | number | The pet's unique record ID. |
+| `name` | string | The pet's name. |
+| `type` | string | The type of pet. <br/> Can be one of: `cat`, `dog` |
+| `breed` | string | The pet's breed. |
+| `age` | string | The pet's age. |
```

Comment:
Updated in https://github.com/syangabq/pet-adoption-service/pull/3 -- this will be a number that represents the pet's age in months.

Reply:
Updated!